docs/SDK_RELEASE_NOTES.md

```
@@ -1,6 +1,12 @@
+# Unreleased
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)